lib/Pages/settings_screen.dart

```diff
@@ -561,245 +565,297 @@ class AuthenticationSection extends StatelessWidget {
+                              )
+                            : Column(
+                                children: [
+                                  SettingsTextField(
+                                    validator: (value) {
+                                      if (value!.isEmpty) {
+                                        return "Host or IP cannot be empty";
+                                      }
+                                      return null;
+                                    },
+                                    hintText: 'Host or IP',
+                                    labelText: 'Host',
+                                    controller: hostController,
+                                  ),
+                                  SizedBox(height: 20),
+                                  SettingsTextField(
+                                    validator: (value) {
+                                      if (value!.isEmpty) {
+                                        return "Port cannot be empty";
+                                      }
+                                      return null;
+                                    },
+                                    hintText: 'Port',
+                                    labelText: 'Port',
+                                    controller: portController,
+                                  )
+                                ],
+                              ),
+                      ],
+                    )
+                  : Column(
+                      children: [
+                        SettingsTextField(
+                          validator: (value) {
+                            if (value!.isEmpty) {
+                              return "Client Username cannot be empty.";
+                            }
+                            return null;
+                          },
+                          hintText: 'Client Username',
+                          labelText: 'Username',
+                          controller: clientUsernameController,
+                        ),
+                        SizedBox(height: 20),
+                        SettingsTextField(
+                          validator: (value) {
+                            if (value!.isEmpty) {
+                              return "Client Password cannot be empty.";
+                            }
+                            return null;
+                          },
+                          hintText: 'Client Password',
+                          labelText: 'Password',
+                          controller: clientPasswordController,
+                        ),
+                        SizedBox(height: 20),
+                        SettingsTextField(
+                          validator: (value) {
+                            if (value!.isEmpty) {
+                              return "URL cannot be empty";
+                            } else if (!value.contains("http://")) {
```

Comment:
This won't verify if the provided string is a valid URL or not since a string like `123http://` will also be accepted. You need to write a RegEx to validate this or you can also try using a flutter package to validate this

Reply:
@virajbhartiya thanks for find this bug. I forgot to notice that. I will fix soon.